frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -64,6 +64,7 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
       lastSelectedSize: notebookSize.name,
       lastSelectedImage: `${image.imageVersion?.from.name}`,
       projectName,
+      notebookName,
```

Comment:
This comes from params -- you're not handling create name. Get it from the `startNotebookData`?